rllib/utils/actor_manager.py

```diff
@@ -696,10 +700,24 @@ def fetch_ready_async_reqs(
             The values may be actual data returned or exceptions raised during the
             remote call in the format of RemoteCallResults.
         """
+        if isinstance(tags, str):
+            tags = {tags}
+        elif isinstance(tags, list):
+            tags = set(tags)
+        else:
+            raise ValueError(
+                f"tags must be either a str or a list of str, got {type(tags)}."
+            )
+        remote_calls = []
+        remote_actor_ids = []
+        for call, (tag, actor_id) in self.__in_flight_req_to_actor_id.items():
+            if tag in tags:
```

Comment:
what if tags is empty?
can you add a unit test for this case as well?

Reply:
of course thank you for catching this.